src/lib/swapper/api.ts

```diff
@@ -34,31 +34,18 @@ export const makeSwapErrorRight = ({
   code,
 })
 
-export type EvmFeeData = {
-  estimatedGasCryptoBaseUnit?: string
-  gasPriceCryptoBaseUnit?: string
-  approvalFeeCryptoBaseUnit?: string
-  maxFeePerGas?: string
-  maxPriorityFeePerGas?: string
-}
-
 export type UtxoFeeData = {
   byteCount: string
   satsPerByte: string
 }
 
 export type CosmosSdkFeeData = {
-  estimatedGasCryptoBaseUnit: string
+  approvalGasLimitCryptoBaseUnit: string
```

Comment:
There is no notion of approvals in Cosmos SDK, though there definitely is a notion of an estimated gas, revert?

Reply:
my mistake ser, fixed